inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/TestFileAgent.java

```diff
@@ -144,6 +146,21 @@ public void testOneJobOnly() throws Exception {
         Assert.assertTrue(checkOnlyOneJob());
     }
 
+    // @Test
+    public void testOneJobFullPath() throws Exception {
+        String jsonString = TestUtils.getTestTriggerProfile();
+        TriggerProfile triggerProfile = TriggerProfile.parseJsonStr(jsonString);
+        triggerProfile.set(JOB_DIR_FILTER_PATTERN,
+                Paths.get(getClass().getClassLoader().getResource("test").toURI()).toString());
+        triggerProfile.set(JOB_FILE_MAX_WAIT, "-1");
+        triggerProfile.set(JOB_FILE_COLLECT_TYPE, FileCollectType.FULL);
+        TriggerManager triggerManager = agent.getManager().getTriggerManager();
+        triggerManager.addTrigger(triggerProfile);
+        while (true) {
```

Comment:
what's the purpose of this code

Reply:
In full mode, you can add files to the target after running to test whether the increment is normal

Reply:
runing a dead loop is not proper

Reply:
It has been modified to add files to the code to test the addition of full mode